src/timing.c

```diff
@@ -165,8 +167,14 @@ JL_DLLEXPORT void jl_timing_show(jl_value_t *v, jl_timing_block_t *cur_block)
 JL_DLLEXPORT void jl_timing_show_module(jl_module_t *m, jl_timing_block_t *cur_block)
 {
 #ifdef USE_TRACY
-    const char *module_name = jl_symbol_name(m->name);
-    TracyCZoneText(*(cur_block->tracy_ctx), module_name, strlen(module_name));
+    jl_module_t *root = jl_module_root(m);
```

Comment:
> and also not print out the root module if it is Main since that felt kind of pointless.

I don't see that implemented here?

Reply:
Oh, I must have missed pushing that. Will update.